src/presets/solid-router.ts

```diff
@@ -2,28 +2,33 @@ import type { ImportsMap } from '../types'
 
 export default <ImportsMap>({
   '@solidjs/router': [
-    'Link',
-    'NavLink',
+    // Components
+    'A',
+    'HashRouter',
+    'MemoryRouter',
     'Navigate',
-    'Outlet',
     'Route',
     'Router',
-    'Routes',
-    '_mergeSearchString',
-    'createIntegration',
-    'hashIntegration',
-    'normalizeIntegration',
-    'pathIntegration',
-    'staticIntegration',
-    'useHref',
+
+    // Data APIs
+    'action',
+    'createAsync',
+    'createAsyncStore',
+    'query',
```

Comment:
maybe `action` & `query` are too general too?

at least we can set `ignore` option so let users decide it